Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -306,7 +306,7 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
     }
     
     /**
-     Initializes a navigation view controller that presents the user interface for following a predefined route based on the given options.
+     Initializes a `NavigationViewController` that presents the user interface for following a predefined route based on the given options.
```

Comment:
By the way, it’s OK to refer to the thing the class represents rather than the symbol. There’s no ambiguity because the developer would read this documentation comment in the context of the NavigationViewController class reference already.